app/server/src/client/preview/index.ts

```diff
@@ -24,11 +24,11 @@ export const storiesOf: ClientApi['storiesOf'] = (kind, m) => {
   });
 };
 
-const setRenderFetchAndConfigure: ClientApi['configure'] = (loader, module, options) => {
-  if (options && options.fetchStoryHtml) {
-    setFetchStoryHtml(options.fetchStoryHtml);
-  }
-  api.configure(loader, module, framework);
+const setRenderFetchAndConfigure: ClientApi['configure'] = (...args) => {
+  // if (args.fetchStoryHtml) {
+  //   setFetchStoryHtml(args.fetchStoryHtml);
+  // }
```

Comment:
I'd like to have an option to customize the fetch implementation so that consumers can line up the fetch with their backend implementation. This needs to be a javascript function that is passed so it can't easily be serialized. @shilman or @tmeasday what are the choices here?

Reply:
@jonspalmer can we use a parameter for this? 

It's OK if a parameter is not serializable, that'll just mean it is not sent over the channel to the manager, but it'll still be available on the preview side.

Reply:
Awesome that totally works. implementation updated and docs added.